linkerd2-proxy/src/rt.rs

```diff
@@ -9,45 +12,74 @@ pub(crate) fn build() -> Runtime {
     //
     // The basic scheduler is used when the threaded scheduler would provide no
     // benefit.
-    let mut cores = std::env::var("LINKERD2_PROXY_CORES")
+
+    let min_cores = std::env::var("LINKERD2_PROXY_CORES_MIN")
```

Comment:
now that this environment variable is no longer respected, and we use these `_MIN`/`_MAX`/`_RATIO` variables instead, should this branch be marked as a breaking change?